Postman/Local Testing.postman_environment.json

```diff
@@ -6,9 +6,14 @@
 			"key": "environment",
 			"value": "localhost:8000",
 			"enabled": true
+		},
+		{
+			"key": "file_location",
+			"value": "\"/sample.json\"",
```

Comment:
I was getting permission errors when trying to save to `/sample.json`. I tried different path variations, and just plain `sample.json` seemed to work for me.

Reply:
- The retrieve_nodes routes works with the request.GET.get('file') if we send the file as a url parameter. However, since the file location is expected to be coming in the request body, using request.GET.get('file') will return none. So to parse from the body I followed this: https://stackoverflow.com/questions/29780060/trying-to-parse-request-body-from-post-in-django 
I'm fine changing this file to a url parameter instead and having it there, my thinking was that since file paths can get pretty lengthy we might not want to have them as a url parameter. But I can also see the point on passing it as a parameter.
- I was thinking about it as a back-end task as well, but my rationale was that nodes are more of an IDE feature than a workflow one. Meaning one might save a workflow but then having the ability to re-use the nodes in other workflows. So the file location for the nodes might not necessarily be the same as the file location of the workflow. Idk if if it makes sense, but I'm happy to hear what you think about that as well as the others. 
- sample.json might not work, I just added a path to have something there, the idea is that you can put any absolute path in there.

Reply:
Is this intended to load a server-side JSON file by name, or accept an upload from the client? [The body of a GET request should be ignored](https://stackoverflow.com/questions/978061/http-get-with-request-body), so if it's the former, it needs to go in the querystring, and if it's the latter it needs to be a POST.